modules/preview-react/breadcrumbs/stories/Breadcrumbs.stories.mdx

```diff
@@ -0,0 +1,219 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+import {Breadcrumbs} from '@workday/canvas-kit-preview-react/breadcrumbs';
+import {Specifications} from '@workday/canvas-kit-docs';
+
+import {StandardList} from './examples/StandardList';
+import {CollapsibleList} from './examples/CollapsibleList';
+import {RTLCollapsibleList} from './examples/RTL';
+import {CurrentItemTruncation} from './examples/CurrentItemTruncation';
+import {LinkTruncation} from './examples/LinkTruncation';
+
+<Meta title="Preview/Breadcrumbs/React" component={Breadcrumbs} />
+
+# Canvas Kit Breadcrumbs
+
+Breadcrumbs allow users to keep track and maintain awareness of their location as they navigate
+through pages, folders, files, etc.
+
+[> Workday Design Reference](https://design.workday.com/components/navigation/breadcrumbs)
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-preview-react
+```
+
+## Usage
+
+Breadcrumbs is a composable component that provides some flexibility in it's structure. There are
+two main variations: standard list and collapsible list. We'll discuss the standard list first. All
+subcomponents are described in more detail in the [Components](#components) section below.
```

Comment:
Yeah, good call out. I initially wrote this paragraph when we were first introducing the idea of composable components. On a second read, I think it feels too close to compound components (which it isn't), and it could be easily confused. I'll remove it. 👍🏻